OpenTabletDriver.UX/Windows/Greeter/Pages/BackgroundingPage.cs

```diff
@@ -0,0 +1,23 @@
+using Eto.Drawing;
+using OpenTabletDriver.UX.Attributes;
+using OpenTabletDriver.UX.Controls.Generic;
+
+namespace OpenTabletDriver.UX.Windows.Greeter.Pages
+{
+    [PageName("Running in the background")]
+    public class BackgroundingPage : StylizedPage
+    {
+        public BackgroundingPage()
+        {
+            this.Content = new StackedContent
+            {
+                new PaddingSpacerItem(),
+                new Bitmap(App.Logo.WithSize(150, 150)),
+                new StylizedText("Running in the background", SystemFonts.Bold(12), new Padding(0, 0, 0, 8)),
+                "You can run OpenTabletDriver in the background.",
+                "All you have to do is minimize it, and it will stay running in your system tray.",
```

Comment:
Maybe better wording this way? The "all you have to do" part implies it's an optional feature and not the standard behavior
```suggestion
                "OpenTabletDriver will run in the background when minimized.",
                "You can show the window again from the system tray.",
```

Reply:
Best wording for this is probably:
```
When minimized, OpenTabletDriver will run in the background.
The window can be restored from the system tray.
```